src/components/tooltip/tooltip.jsx

```diff
@@ -178,7 +178,7 @@ class Tooltip extends React.Component {
 }
 
 Tooltip.defaultProps = {
-  children: <Questionmark fill="#00A9EC" stroke="#00A9EC" width={16} height={16} />,
```

Comment:
This will visually break onboarding process

Reply:
After discussion it seems like this change works there too! Will get some input from design regarding hover state feedback and get back to you!